<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGoods_window</class>
 <widget class="QDialog" name="AddGoods_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="add_to_cart">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>輸入數量一併加入購物車</string>
   </property>
  </widget>
  <widget class="QLabel" name="shop_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>940</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>shop_title</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>960</width>
     <height>270</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="pic_11">
       <property name="text">
        <string>pic_1</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="pic_12">
       <property name="text">
        <string>pic_2</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="pic_13">
       <property name="text">
        <string>pic_3</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="pic_14">
       <property name="text">
        <string>pic_4</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="pic_15">
       <property name="text">
        <string>pic_5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>941</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="1">
      <widget class="QPushButton" name="info_11">
       <property name="text">
        <string>卡片詳細</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="name_11">
       <property name="text">
        <string>name_4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string> 數量 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_12"/>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="info_12">
       <property name="text">
        <string>卡片詳細</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="name_12">
       <property name="text">
        <string>name_3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string> 數量 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_13"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="name_13">
       <property name="text">
        <string>name_2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string> 數量 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_14"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="info_13">
       <property name="text">
        <string>卡片詳細</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="info_14">
       <property name="text">
        <string>卡片詳細</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string> 數量 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_15"/>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string> 數量 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_16"/>
       </item>
      </layout>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="info_15">
       <property name="text">
        <string>卡片詳細</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="name_14">
       <property name="text">
        <string>name_5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="name_15">
       <property name="text">
        <string>name_1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="how_many">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>160</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
    </font>
   </property>
   <property name="text">
    <string>how_many</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_page">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>下10件商品</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>129</y>
     <width>941</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="up_gridLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
